tardis/plasma/tests/test_nlte_excitation.py

```diff
@@ -112,11 +113,13 @@ def test_prepare_bound_bound_rate_matrix(
     ]
     # if this test fails the first thing to check is if the reshape in the
     # methods made a view or a copy. If it's a copy rewrite the function.
-    assert_allclose(
-        desired_rate_matrix,
-        np.array(actual_rate_matrix),
-        rtol=1e-6,
-    )
+    # TODO: allow rtol=1e-6
+    assert snapshot_np == np.array(actual_rate_matrix)
+    # assert_allclose(
```

Comment:
Does assert_allclose not work with the snapshot?

Reply:
I'd have to create another snapshot fixture to get that to work. Does `assert_almostequal` not fulfill the needs?

Reply:
The Syrupy devs are working on another PR which would enable the functionality to do this- 
https://github.com/tophat/syrupy/pull/816

Reply:
That's good, but this functionality is critical for our tests. I'm concerned about the timeline for the syrupy updates in this case.

Reply:
I would ask them.